tests/_disable_chrome_installation.pm

```diff
@@ -0,0 +1,26 @@
+# vi: set shiftwidth=4 tabstop=4 expandtab:
+use base 'installedtest';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    # Disable automatic installation of Google Chrome. It pops up authentication
```

Comment:
IIRC you had some trouble getting this to work – what was the problem, or is it still intermittently failing?

Reply:
It’s still failing, I think. At least, *something* is popping up polkit authentication dialogues shortly after logging in. The PR which was merged recently to bump up the polkit logging should help debug this, but I’d like to get this PR in before taking another look at the Chrome issue, since it’s slightly orthogonal.